pages/advanced/content/theming.md

```diff
@@ -1,3 +1,122 @@
 ---
 title: Theming
 ---
+
+`glamorous` fully supports theming using a special `<ThemeProvider>` component.
+
+It provides the `theme` to all glamorous components down the tree.
+
+> Try this out in your browser [here](https://codesandbox.io/s/o2yq9MkQk)!
+
+```source-js-jsx
```

Comment:
Haha! This came from `refined-github`! I know because I just _barely_ experienced the same thing 60 seconds ago 😄 

Change this to `js` 😉